Gemfile

```diff
@@ -8,4 +8,4 @@ gem 'rspec'
 gem 'webmock'
 gem 'pry-byebug'
 gem 'faraday-retry'
-gem 'colorize'
\ No newline at end of file
+gem 'colorize'
```

Comment:
(Nitpick) I think I would go back and amend the commit where this line was added to include the trailing newline, instead of adding the newline in a later commit

Reply:
I have now made this change on the 1st commit.